datadog/resource_datadog_dashboard_list.go

```
@@ -49,102 +49,117 @@ func resourceDatadogDashboardList() *schema.Resource {
 }
 
 func resourceDatadogDashboardListCreate(d *schema.ResourceData, meta interface{}) error {
+	providerConf := meta.(*ProviderConfiguration)
+	client := providerConf.CommunityClient
+
 	dashboardList, err := buildDatadogDashboardList(d)
 	if err != nil {
 		fmt.Printf("Error building the dashboard list %s", err.Error())
```

As discussed in person, we should return the error here.